apistructs/addon.go

```diff
@@ -306,7 +306,7 @@ const (
 	// AddonConfigCenterAlias terminus-configcenter alias name
 	AddonConfigCenterAlias = "configcenter"
 	// AddonNewConfigCenter configcenter
-	AddonNewConfigCenter = "configcenter"
+	AddonNewConfigCenter = "config-center"
```

Comment:
why change `configcenter` to `config-center`?
but still `registercenter`, not `register-center`.